components/status_dropdown/status_dropdown.jsx

```diff
@@ -151,31 +136,30 @@ export default class StatusDropdown extends React.PureComponent {
     }
     handleClearStatus = (e) => {
         e.stopPropagation();
+        e.preventDefault();
         this.props.actions.unsetCustomStatus();
     };
 
-    onToggle = (open) => {
-        this.props.actions.setStatusDropdown(open);
-    }
+    onToggle = (open) => this.props.actions.setStatusDropdown(open);
 
     renderCustomStatus = () => {
         if (!this.props.isCustomStatusEnabled) {
             return null;
         }
-        let customStatusText = localizeMessage('status_dropdown.set_custom', 'Set a Custom Status');
-        let customStatusEmoji = <EmojiIcon className={'custom-status-emoji'}/>;
         const customStatus = this.props.customStatus;
         const isStatusSet = customStatus && (customStatus.text || customStatus.emoji);
-        if (isStatusSet) {
-            customStatusEmoji = (
+        const customStatusText = isStatusSet ? customStatus.text : localizeMessage('status_dropdown.set_custom', 'Set a Custom Status');
+        const customStatusEmoji = isStatusSet ?
+            (
                 <span className='d-flex'>
                     <CustomStatusEmoji
                         showTooltip={false}
+                        emojiStyle={{}}
```

Comment:
Should this be an optional prop?

Reply:
Yes, empty object is passed here to avoid applying default styling.... If it becomes a required prop we would have to pass styling everywhere

Reply:
Update this as discussed.